### ISSUE TITLE
Email notifications to specific addresses based on country

### ISSUE THREAD

**Issue by maneesha** [swcarpentry/amy#1368]
For our workshop request form, can we have it set up to conditionally email a different address (or list of addresses) based on country?  Currently emails go to the admin-all list which means people get emails that are not relevant to them.

**Comment by pbanaszkiewicz** [comment]
Yes, we can do that.

**Comment by pbanaszkiewicz** [comment]
@maneesha:
>  How difficult will it be to update this in the future if we want to change countries and/or emails?

It makes this issue interesting ;-) I can add an admin interface for notification filtering (say, something similar but less extensive than [Thunderbird message filtering](https://support.mozilla.org/en-US/kb/organize-your-messages-using-filters)).

Matching by "Africa" would probably require [migration](https://github.com/swcarpentry/amy/issues/1327#issuecomment-422542068) to [`django-countries-plus`](https://github.com/cordery/django-countries-plus), as it covers more demographic data (incl. continent) than just country name and its ISO code.

@maneesha do you want this only for the [new unified form](https://github.com/swcarpentry/amy/issues/1344), or for all forms (except profile update request)?

**Comment by maneesha** [comment]
What other forms do we have?
* New unified request form - yes, email by country
* Training applications - no notifications

I don't believe we have any other actively used forms.  

Can we keep matching by Africa to include all countries in [this list](https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_Africa)?  

For now I thought we could do something like 
```
if country=="UK":
   mailto = admin-uk
if country in ["EG", "LY", "DZ" .... ]
    mailto = admin-afr
# and so on
```

If it's easier to use the `django-countries-plus` model, I'll leave that to you.

**Comment by maneesha** [comment]
> @maneesha:
> 
> 1. For now we still have Workshop Request and DC Self-Organized Workshop Request forms.

The new unified form should take the place of these forms.

> 2. 54 states in Africa, some of them may be in our country list, some may not; I'd rather outsource the matching to someone else (django-countries-plus), but that's just me being lazy.

I will leave this up to you - whatever would be easiest :-).

**Comment by pbanaszkiewicz** [comment]
@maneesha I'm running a little behind the schedule on this one. Is it okay for you if I continued the work on it tomorrow and shipped the v2.2 on Friday?

**Comment by maneesha** [comment]
No problem!
